# Production Company Analysis

### Jordan - 6/4

Analyzing top 10 production companies (and their divisions) based on https://reelrundown.com/film-industry/Top-10-Movie-Production-Companies 

In [3]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

In [4]:
movies = pd.read_csv("/Users/jordan_uyeki/Documents/ucla/year4/spring2019/Stats131/Project/the-movies-dataset/movies_metadata.csv")
movies.head()

/Users/jordan_uyeki/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py:2785: DtypeWarning: Columns (10) have mixed types. Specify dtype option on import or set low_memory=False.
  interactivity=interactivity, compiler=compiler, result=result)


adult                              belongs_to_collection    budget  \
0  False  {'id': 10194, 'name': 'Toy Story Collection', ...  30000000   
1  False                                                NaN  65000000   
2  False  {'id': 119050, 'name': 'Grumpy Old Men Collect...         0   
3  False                                                NaN  16000000   
4  False  {'id': 96871, 'name': 'Father of the Bride Col...         0   

                                              genres  \
0  [{'id': 16, 'name': 'Animation'}, {'id': 35, '...   
1  [{'id': 12, 'name': 'Adventure'}, {'id': 14, '...   
2  [{'id': 10749, 'name': 'Romance'}, {'id': 35, ...   
3  [{'id': 35, 'name': 'Comedy'}, {'id': 18, 'nam...   
4                     [{'id': 35, 'name': 'Comedy'}]   

                               homepage     id    imdb_id original_language  \
0  http://toystory.disney.com/toy-story    862  tt0114709                en   
1                                   NaN   8844  tt0113497                en   
2                                   NaN  15602  tt0113228                en   
3                                   NaN  31357  tt0114885                en   
4                                   NaN  11862  tt0113041                en   

                original_title  \
0                    Toy Story   
1                      Jumanji   
2             Grumpier Old Men   
3            Waiting to Exhale   
4  Father of the Bride Part II   

                                            overview    ...     release_date  \
0  Led by Woody, Andy's toys live happily in his ...    ...       1995-10-30   
1  When siblings Judy and Peter discover an encha...    ...       1995-12-15   
2  A family wedding reignites the ancient feud be...    ...       1995-12-22   
3  Cheated on, mistreated and stepped on, the wom...    ...       1995-12-22   
4  Just when George Banks has recovered from his ...    ...       1995-02-10   

       revenue runtime                                   spoken_languages  \
0  373554033.0    81.0           [{'iso_639_1': 'en', 'name': 'English'}]   
1  262797249.0   104.0  [{'iso_639_1': 'en', 'name': 'English'}, {'iso...   
2          0.0   101.0           [{'iso_639_1': 'en', 'name': 'English'}]   
3   81452156.0   127.0           [{'iso_639_1': 'en', 'name': 'English'}]   
4   76578911.0   106.0           [{'iso_639_1': 'en', 'name': 'English'}]   

     status                                            tagline  \
0  Released                                                NaN   
1  Released          Roll the dice and unleash the excitement!   
2  Released  Still Yelling. Still Fighting. Still Ready for...   
3  Released  Friends are the people who let you be yourself...   
4  Released  Just When His World Is Back To Normal... He's ...   

                         title  video vote_average vote_count  
0                    Toy Story  False          7.7     5415.0  
1                      Jumanji  False          6.9     2413.0  
2             Grumpier Old Men  False          6.5       92.0  
3            Waiting to Exhale  False          6.1       34.0  
4  Father of the Bride Part II  False          5.7      173.0  

[5 rows x 24 columns]

In [5]:
movies.columns

Index(['adult', 'belongs_to_collection', 'budget', 'genres', 'homepage', 'id',
       'imdb_id', 'original_language', 'original_title', 'overview',
       'popularity', 'poster_path', 'production_companies',
       'production_countries', 'release_date', 'revenue', 'runtime',
       'spoken_languages', 'status', 'tagline', 'title', 'video',
       'vote_average', 'vote_count'],
      dtype='object')

In [6]:
movies.shape

(45466, 24)

In [138]:
movies = movies.sort_values(by = 'revenue', ascending = False)

In [139]:
prod_companies = movies[pd.notnull(movies['production_companies'])]
prod_companies = prod_companies.sort_values(by = 'revenue', ascending = False)
prod_companies['belongs_to_series'] = prod_companies.belongs_to_collection.isna()

##  #1 Warner Bros.

In [140]:
warner_bros = prod_companies[prod_companies['production_companies'].str.contains("Warner Bros.|New Line Cinema|Castle Rock|Turner Entertainment|DC Films")]
warner_bros.head()

adult                              belongs_to_collection     budget  \
17437  False  {'id': 1241, 'name': 'Harry Potter Collection'...  125000000   
7000   False  {'id': 119, 'name': 'The Lord of the Rings Col...   94000000   
18252  False  {'id': 263, 'name': 'The Dark Knight Collectio...  250000000   
19971  False  {'id': 121938, 'name': 'The Hobbit Collection'...  250000000   
12481  False  {'id': 263, 'name': 'The Dark Knight Collectio...  185000000   

                                                  genres  \
17437  [{'id': 10751, 'name': 'Family'}, {'id': 14, '...   
7000   [{'id': 12, 'name': 'Adventure'}, {'id': 14, '...   
18252  [{'id': 28, 'name': 'Action'}, {'id': 80, 'nam...   
19971  [{'id': 12, 'name': 'Adventure'}, {'id': 14, '...   
12481  [{'id': 18, 'name': 'Drama'}, {'id': 28, 'name...   

                                           homepage     id    imdb_id  \
17437                    http://www.harrypotter.com  12445  tt1201607   
7000                  http://www.lordoftherings.net    122  tt0167260   
18252            http://www.thedarkknightrises.com/  49026  tt1345836   
19971                     http://www.thehobbit.com/  49051  tt0903624   
12481  http://thedarkknight.warnerbros.com/dvdsite/    155  tt0468569   

      original_language                                 original_title  \
17437                en   Harry Potter and the Deathly Hallows: Part 2   
7000                 en  The Lord of the Rings: The Return of the King   
18252                en                          The Dark Knight Rises   
19971                en              The Hobbit: An Unexpected Journey   
12481                en                                The Dark Knight   

                                                overview        ...         \
17437  Harry, Ron and Hermione continue their quest t...        ...          
7000   Aragorn is revealed as the heir to the ancient...        ...          
18252  Following the death of District Attorney Harve...        ...          
19971  Bilbo Baggins, a hobbit enjoying his quiet lif...        ...          
12481  Batman raises the stakes in his war on crime. ...        ...          

            revenue runtime  \
17437  1.342000e+09   130.0   
7000   1.118889e+09   201.0   
18252  1.084939e+09   165.0   
19971  1.021104e+09   169.0   
12481  1.004558e+09   152.0   

                                        spoken_languages    status  \
17437           [{'iso_639_1': 'en', 'name': 'English'}]  Released   
7000            [{'iso_639_1': 'en', 'name': 'English'}]  Released   
18252           [{'iso_639_1': 'en', 'name': 'English'}]  Released   
19971           [{'iso_639_1': 'en', 'name': 'English'}]  Released   
12481  [{'iso_639_1': 'en', 'name': 'English'}, {'iso...  Released   

                                                 tagline  \
17437                                  It all ends here.   
7000                     The eye of the enemy is moving.   
18252                                    The Legend Ends   
19971  From the smallest beginnings come the greatest...   
12481                                    Why So Serious?   

                                               title  video vote_average  \
17437   Harry Potter and the Deathly Hallows: Part 2  False          7.9   
7000   The Lord of the Rings: The Return of the King  False          8.1   
18252                          The Dark Knight Rises  False          7.6   
19971              The Hobbit: An Unexpected Journey  False          7.0   
12481                                The Dark Knight  False          8.3   

      vote_count belongs_to_series  
17437     6141.0             False  
7000      8226.0             False  
18252     9263.0             False  
19971     8427.0             False  
12481    12269.0             False  

[5 rows x 25 columns]

In [141]:
warner_bros.shape

(1711, 25)

In [142]:
warner_bros.belongs_to_series.value_counts()

True     1434
False     277
Name: belongs_to_series, dtype: int64

In [143]:
warner_bros = warner_bros.sort_values(by = 'revenue', ascending = False)
warner_bros = warner_bros.iloc[1:101,];
warner_bros.belongs_to_series.value_counts()

False    65
True     35
Name: belongs_to_series, dtype: int64

Out of all of the movies produced by the Warner Bros, 19.3% of the movies belonged to a series.  Out of the top 100 movies produced by the Warner Bros, 65% of the movies belonged to a series. 

##  #2 Sony Pictures

In [144]:
sony_pictures = prod_companies[prod_companies['production_companies'].str.contains("Sony|TriStar Pictures|Screen Gems|Destination Films|Affirm Films")]
sony_pictures.head()

adult                              belongs_to_collection     budget  \
17498  False  {'id': 134897, 'name': 'The Smurfs (Theatrical...  110000000   
33246  False  {'id': 185103, 'name': 'Hotel Transylvania Col...   80000000   
1773   False                                                NaN  130000000   
19569  False  {'id': 185103, 'name': 'Hotel Transylvania Col...   85000000   
1060   False  {'id': 86336, 'name': 'Basic Instinct Collecti...   49000000   

                                                  genres  \
17498  [{'id': 16, 'name': 'Animation'}, {'id': 10751...   
33246  [{'id': 16, 'name': 'Animation'}, {'id': 35, '...   
1773   [{'id': 878, 'name': 'Science Fiction'}, {'id'...   
19569  [{'id': 16, 'name': 'Animation'}, {'id': 35, '...   
1060   [{'id': 53, 'name': 'Thriller'}, {'id': 9648, ...   

                                                homepage      id    imdb_id  \
17498                       http://www.smurfhappens.com/   41513  tt0472181   
33246                                                NaN  159824  tt2510894   
1773                                                 NaN     929  tt0120685   
19569                     http://www.welcometohotelt.com   76492  tt0837562   
1060   https://www.sonymoviechannel.com/movies/basic-...     402  tt0103772   

      original_language        original_title  \
17498                en            The Smurfs   
33246                en  Hotel Transylvania 2   
1773                 en              Godzilla   
19569                en    Hotel Transylvania   
1060                 en        Basic Instinct   

                                                overview        ...         \
17498  When the evil wizard Gargamel chases the tiny ...        ...          
33246  When the old-old-old-fashioned vampire Vlad ar...        ...          
1773   When a freighter is viciously attacked in the ...        ...          
19569  Dracula, who operates a high-end resort away f...        ...          
1060   A police detective is in charge of the investi...        ...          

           revenue runtime                                   spoken_languages  \
17498  563749323.0   103.0           [{'iso_639_1': 'en', 'name': 'English'}]   
33246  473226958.0    89.0  [{'iso_639_1': 'en', 'name': 'English'}, {'iso...   
1773   379014294.0   139.0  [{'iso_639_1': 'en', 'name': 'English'}, {'iso...   
19569  358375603.0    91.0           [{'iso_639_1': 'en', 'name': 'English'}]   
1060   352927224.0   127.0           [{'iso_639_1': 'en', 'name': 'English'}]   

         status                                            tagline  \
17498  Released                                     Smurf Happens.   
33246  Released              They're back to raise a little terror   
1773   Released                                  Size does matter.   
19569  Released          Where monsters go to get away from it all   
1060   Released  A brutal murder. A brilliant killer. A cop who...   

                      title  video vote_average vote_count belongs_to_series  
17498            The Smurfs  False          5.5     1201.0             False  
33246  Hotel Transylvania 2  False          6.7     1528.0             False  
1773               Godzilla  False          5.3     1075.0              True  
19569    Hotel Transylvania  False          6.8     2622.0             False  
1060         Basic Instinct  False          6.5      869.0             False  

[5 rows x 25 columns]

In [145]:
sony_pictures.shape

(448, 25)

In [75]:
sony_pictures.belongs_to_series.value_counts()

True     359
False     89
Name: belongs_to_series, dtype: int64

In [76]:
sony_pictures = sony_pictures.sort_values(by = 'revenue', ascending = False)
sony_pictures = sony_pictures.iloc[1:101,];
sony_pictures.belongs_to_series.value_counts()

True     69
False    31
Name: belongs_to_series, dtype: int64

Out of all of the movies produced by Sony Pictures, 19.9% of the movies belonged to a series.
  Out of the top 100 movies produced by Sony Pictures, 31% of the movies belonged to a series.

##  #3 Walt Disney Studios

In [78]:
walt_disney = prod_companies[prod_companies['production_companies'].str.contains("Disney|Lucas Film|Pixar|Marvel Studios")]
walt_disney.head()

adult                              belongs_to_collection     budget  \
17818  False  {'id': 86311, 'name': 'The Avengers Collection...  220000000   
26558  False  {'id': 86311, 'name': 'The Avengers Collection...  280000000   
22110  False  {'id': 386382, 'name': 'Frozen Collection', 'p...  150000000   
42222  False                                                NaN  160000000   
20830  False  {'id': 131292, 'name': 'Iron Man Collection', ...  200000000   

                                                  genres  \
17818  [{'id': 878, 'name': 'Science Fiction'}, {'id'...   
26558  [{'id': 28, 'name': 'Action'}, {'id': 12, 'nam...   
22110  [{'id': 16, 'name': 'Animation'}, {'id': 12, '...   
42222  [{'id': 10751, 'name': 'Family'}, {'id': 14, '...   
20830  [{'id': 28, 'name': 'Action'}, {'id': 12, 'nam...   

                                                homepage      id    imdb_id  \
17818                  http://marvel.com/avengers_movie/   24428  tt0848228   
26558  http://marvel.com/movies/movie/193/avengers_ag...   99861  tt2395427   
22110                    http://movies.disney.com/frozen  109445  tt2294629   
42222  http://movies.disney.com/beauty-and-the-beast-...  321612  tt2771200   
20830                         http://marvel.com/ironman3   68721  tt1300854   

      original_language           original_title  \
17818                en             The Avengers   
26558                en  Avengers: Age of Ultron   
22110                en                   Frozen   
42222                en     Beauty and the Beast   
20830                en               Iron Man 3   

                                                overview        ...         \
17818  When an unexpected enemy emerges and threatens...        ...          
26558  When Tony Stark tries to jumpstart a dormant p...        ...          
22110  Young princess Anna of Arendelle dreams about ...        ...          
42222  A live-action adaptation of Disney's version o...        ...          
20830  When Tony Stark's world is torn apart by a for...        ...          

            revenue runtime                          spoken_languages  \
17818  1.519558e+09   143.0  [{'iso_639_1': 'en', 'name': 'English'}]   
26558  1.405404e+09   141.0  [{'iso_639_1': 'en', 'name': 'English'}]   
22110  1.274219e+09   102.0  [{'iso_639_1': 'en', 'name': 'English'}]   
42222  1.262886e+09   129.0  [{'iso_639_1': 'en', 'name': 'English'}]   
20830  1.215440e+09   130.0  [{'iso_639_1': 'en', 'name': 'English'}]   

         status                                            tagline  \
17818  Released                            Some assembly required.   
26558  Released                                A New Age Has Come.   
22110  Released  Only the act of true love will thaw a frozen h...   
42222  Released                                      Be our guest.   
20830  Released                Unleash the power behind the armor.   

                         title  video vote_average vote_count  \
17818             The Avengers  False          7.4    12000.0   
26558  Avengers: Age of Ultron  False          7.3     6908.0   
22110                   Frozen  False          7.3     5440.0   
42222     Beauty and the Beast  False          6.8     5530.0   
20830               Iron Man 3  False          6.8     8951.0   

      belongs_to_series  
17818             False  
26558             False  
22110             False  
42222              True  
20830             False  

[5 rows x 25 columns]

In [79]:
walt_disney.shape

(564, 25)

In [80]:
walt_disney.belongs_to_series.value_counts()

True     376
False    188
Name: belongs_to_series, dtype: int64

In [81]:
walt_disney = walt_disney.sort_values(by = 'revenue', ascending = False)
walt_disney = walt_disney.iloc[1:101,];
walt_disney.belongs_to_series.value_counts()

False    60
True     40
Name: belongs_to_series, dtype: int64

Out of all of the movies produced by Walt Disney Studios, 33.3% of the movies belonged to a series. Out of the top 100 movies produced by Walt Disney Studios, 60% of the movies belonged to a series.

## #4 Universal Picturses

In [86]:
universal_pictures = prod_companies[prod_companies['production_companies'].str.contains("Universal Studios|DreamWorks|Focus Features|Working Title Films")]
universal_pictures.head()

adult                              belongs_to_collection     budget  \
25084  False  {'id': 328, 'name': 'Jurassic Park Collection'...  150000000   
7717   False  {'id': 2150, 'name': 'Shrek Collection', 'post...  150000000   
13833  False  {'id': 8650, 'name': 'Transformers Collection'...  150000000   
11825  False  {'id': 2150, 'name': 'Shrek Collection', 'post...  160000000   
20970  False  {'id': 9485, 'name': 'The Fast and the Furious...  160000000   

                                                  genres  \
25084  [{'id': 28, 'name': 'Action'}, {'id': 12, 'nam...   
7717   [{'id': 12, 'name': 'Adventure'}, {'id': 16, '...   
13833  [{'id': 878, 'name': 'Science Fiction'}, {'id'...   
11825  [{'id': 14, 'name': 'Fantasy'}, {'id': 12, 'na...   
20970  [{'id': 28, 'name': 'Action'}, {'id': 53, 'nam...   

                                            homepage      id    imdb_id  \
25084                  http://www.jurassicworld.com/  135397  tt0369610   
7717                          http://www.shrek2.com/     809  tt0298148   
13833              http://www.transformersmovie.com/    8373  tt1055369   
11825  http://www.shrekthethird.com/flash/index.html     810  tt0413267   
20970           http://www.thefastandthefurious.com/   82992  tt1905041   

      original_language                       original_title  \
25084                en                       Jurassic World   
7717                 en                              Shrek 2   
13833                en  Transformers: Revenge of the Fallen   
11825                en                      Shrek the Third   
20970                en                     Fast & Furious 6   

                                                overview        ...         \
25084  Twenty-two years after the events of Jurassic ...        ...          
7717   Shrek, Fiona and Donkey set off to Far, Far Aw...        ...          
13833  Sam Witwicky leaves the Autobots behind for a ...        ...          
11825  The King of Far Far Away has died and Shrek an...        ...          
20970  Hobbs has Dominic and Brian reassemble their c...        ...          

            revenue runtime  \
25084  1.513529e+09   124.0   
7717   9.198388e+08    93.0   
13833  8.362972e+08   150.0   
11825  7.989582e+08    93.0   
20970  7.886798e+08   130.0   

                                        spoken_languages    status  \
25084           [{'iso_639_1': 'en', 'name': 'English'}]  Released   
7717            [{'iso_639_1': 'en', 'name': 'English'}]  Released   
13833  [{'iso_639_1': 'en', 'name': 'English'}, {'iso...  Released   
11825           [{'iso_639_1': 'en', 'name': 'English'}]  Released   
20970  [{'iso_639_1': 'en', 'name': 'English'}, {'iso...  Released   

                         tagline                                title  video  \
25084          The park is open.                       Jurassic World  False   
7717   Once upon another time...                              Shrek 2  False   
13833         Revenge is coming.  Transformers: Revenge of the Fallen  False   
11825    Who's ready for Thirds?                      Shrek the Third  False   
20970     All roads lead to this                     Fast & Furious 6  False   

      vote_average vote_count belongs_to_series  
25084          6.5     8842.0             False  
7717           6.7     3082.0             False  
13833          6.0     3192.0             False  
11825          6.0     2355.0             False  
20970          6.7     5282.0             False  

[5 rows x 25 columns]

In [87]:
universal_pictures.shape

(339, 25)

In [88]:
universal_pictures.belongs_to_series.value_counts()

True     283
False     56
Name: belongs_to_series, dtype: int64

In [89]:
universal_pictures = universal_pictures.sort_values(by = 'revenue', ascending = False)
universal_pictures = universal_pictures.iloc[1:101,];
universal_pictures.belongs_to_series.value_counts()

True     70
False    30
Name: belongs_to_series, dtype: int64

Out of all of the movies produced by Universal Pictures, 16.5% of the movies belonged to a series. Out of the top 100 movies produced by Universal Pictures, 30% of the movies belonged to a series.

## #5 20th Century Fox

In [90]:
century_fox = prod_companies[prod_companies['production_companies'].str.contains("Fox|fox")];
century_fox.head()

adult                              belongs_to_collection     budget  \
14551  False  {'id': 87096, 'name': 'Avatar Collection', 'po...  237000000   
1639   False                                                NaN  200000000   
13848  False  {'id': 8354, 'name': 'Ice Age Collection', 'po...   90000000   
19139  False  {'id': 8354, 'name': 'Ice Age Collection', 'po...   95000000   
760    False  {'id': 304378, 'name': 'Independence Day Colle...   75000000   

                                                  genres  \
14551  [{'id': 28, 'name': 'Action'}, {'id': 12, 'nam...   
1639   [{'id': 18, 'name': 'Drama'}, {'id': 10749, 'n...   
13848  [{'id': 16, 'name': 'Animation'}, {'id': 35, '...   
19139  [{'id': 16, 'name': 'Animation'}, {'id': 35, '...   
760    [{'id': 28, 'name': 'Action'}, {'id': 12, 'nam...   

                                                homepage     id    imdb_id  \
14551                        http://www.avatarmovie.com/  19995  tt0499549   
1639                         http://www.titanicmovie.com    597  tt0120338   
13848  http://www.iceagemovies.com/films/ice-age-dawn...   8355  tt1080016   
19139  http://www.iceagemovies.com/films/ice-age-cont...  57800  tt1667889   
760                                                  NaN    602  tt0116629   

      original_language                  original_title  \
14551                en                          Avatar   
1639                 en                         Titanic   
13848                en  Ice Age: Dawn of the Dinosaurs   
19139                en      Ice Age: Continental Drift   
760                  en                Independence Day   

                                                overview        ...         \
14551  In the 22nd century, a paraplegic Marine is di...        ...          
1639   84 years later, a 101-year-old woman named Ros...        ...          
13848  Times are changing for Manny the moody mammoth...        ...          
19139  Manny, Diego, and Sid embark upon another adve...        ...          
760    On July 2, a giant alien mothership enters orb...        ...          

            revenue runtime  \
14551  2.787965e+09   162.0   
1639   1.845034e+09   194.0   
13848  8.866868e+08    94.0   
19139  8.772448e+08    88.0   
760    8.169693e+08   145.0   

                                        spoken_languages    status  \
14551  [{'iso_639_1': 'en', 'name': 'English'}, {'iso...  Released   
1639   [{'iso_639_1': 'en', 'name': 'English'}, {'iso...  Released   
13848           [{'iso_639_1': 'en', 'name': 'English'}]  Released   
19139           [{'iso_639_1': 'en', 'name': 'English'}]  Released   
760             [{'iso_639_1': 'en', 'name': 'English'}]  Released   

                                                 tagline  \
14551                        Enter the World of Pandora.   
1639           Nothing on Earth could come between them.   
13848                        You Won't Believe Your Ice!   
19139  Manny, Diego, and Sid embark upon another adve...   
760      Earth. Take a good look. It might be your last.   

                                title  video vote_average vote_count  \
14551                          Avatar  False          7.2    12114.0   
1639                          Titanic  False          7.5     7770.0   
13848  Ice Age: Dawn of the Dinosaurs  False          6.5     2330.0   
19139      Ice Age: Continental Drift  False          6.2     2731.0   
760                  Independence Day  False          6.7     3334.0   

      belongs_to_series  
14551             False  
1639               True  
13848             False  
19139             False  
760               False  

[5 rows x 25 columns]

In [91]:
century_fox.shape

(1206, 25)

In [92]:
century_fox.belongs_to_series.value_counts()

True     989
False    217
Name: belongs_to_series, dtype: int64

In [93]:
century_fox = century_fox.sort_values(by = 'revenue', ascending = False)
century_fox = century_fox.iloc[1:101,];
century_fox.belongs_to_series.value_counts()

False    55
True     45
Name: belongs_to_series, dtype: int64

Out of all of the movies produced by 20th Century Fox, 18.0% of the movies belonged to a series. Out of the top 100 movies produced by 20th Century Fox, 55% of the movies belonged to a series.

## #6 Paramount Pictures

In [94]:
paramount = prod_companies[prod_companies['production_companies'].str.contains("Paramount|Nickelodean|MTV")];
paramount.head()

adult                              belongs_to_collection     budget  \
1639   False                                                NaN  200000000   
17818  False  {'id': 86311, 'name': 'The Avengers Collection...  220000000   
17293  False  {'id': 8650, 'name': 'Transformers Collection'...  195000000   
23617  False  {'id': 8650, 'name': 'Transformers Collection'...  210000000   
13833  False  {'id': 8650, 'name': 'Transformers Collection'...  150000000   

                                                  genres  \
1639   [{'id': 18, 'name': 'Drama'}, {'id': 10749, 'n...   
17818  [{'id': 878, 'name': 'Science Fiction'}, {'id'...   
17293  [{'id': 28, 'name': 'Action'}, {'id': 878, 'na...   
23617  [{'id': 878, 'name': 'Science Fiction'}, {'id'...   
13833  [{'id': 878, 'name': 'Science Fiction'}, {'id'...   

                                homepage     id    imdb_id original_language  \
1639         http://www.titanicmovie.com    597  tt0120338                en   
17818  http://marvel.com/avengers_movie/  24428  tt0848228                en   
17293  http://www.transformersmovie.com/  38356  tt1399103                en   
23617   http://www.transformersmovie.com  91314  tt2109248                en   
13833  http://www.transformersmovie.com/   8373  tt1055369                en   

                            original_title  \
1639                               Titanic   
17818                         The Avengers   
17293       Transformers: Dark of the Moon   
23617      Transformers: Age of Extinction   
13833  Transformers: Revenge of the Fallen   

                                                overview        ...         \
1639   84 years later, a 101-year-old woman named Ros...        ...          
17818  When an unexpected enemy emerges and threatens...        ...          
17293  Sam Witwicky takes his first tenuous steps int...        ...          
23617  As humanity picks up the pieces, following the...        ...          
13833  Sam Witwicky leaves the Autobots behind for a ...        ...          

            revenue runtime  \
1639   1.845034e+09   194.0   
17818  1.519558e+09   143.0   
17293  1.123747e+09   154.0   
23617  1.091405e+09   165.0   
13833  8.362972e+08   150.0   

                                        spoken_languages    status  \
1639   [{'iso_639_1': 'en', 'name': 'English'}, {'iso...  Released   
17818           [{'iso_639_1': 'en', 'name': 'English'}]  Released   
17293           [{'iso_639_1': 'en', 'name': 'English'}]  Released   
23617           [{'iso_639_1': 'en', 'name': 'English'}]  Released   
13833  [{'iso_639_1': 'en', 'name': 'English'}, {'iso...  Released   

                                                 tagline  \
1639           Nothing on Earth could come between them.   
17818                            Some assembly required.   
17293  The invasion we always feared. An enemy we nev...   
23617                  This is not war. It's extinction.   
13833                                 Revenge is coming.   

                                     title  video vote_average vote_count  \
1639                               Titanic  False          7.5     7770.0   
17818                         The Avengers  False          7.4    12000.0   
17293       Transformers: Dark of the Moon  False          6.1     3351.0   
23617      Transformers: Age of Extinction  False          5.8     3136.0   
13833  Transformers: Revenge of the Fallen  False          6.0     3192.0   

      belongs_to_series  
1639               True  
17818             False  
17293             False  
23617             False  
13833             False  

[5 rows x 25 columns]

In [95]:
paramount.shape

(1081, 25)

In [97]:
paramount.belongs_to_series.value_counts()

True     927
False    154
Name: belongs_to_series, dtype: int64

In [98]:
paramount = paramount.sort_values(by = 'revenue', ascending = False)
paramount = paramount.iloc[1:101,];
paramount.belongs_to_series.value_counts()

False    51
True     49
Name: belongs_to_series, dtype: int64

Out of all of the movies produced by Paramount Pictures, 14.2% of the movies belonged to a series. Out of the top 100 movies produced by Paramount Pictures, 51% of the movies belonged to a series.

## #7 Lionsgate Films

In [105]:
lionsgate = prod_companies[prod_companies['production_companies'].str.contains("Lionsgate|Roadside Attractions|summmit|Summit|lionsgate|Pantelion|Codeblack")];
lionsgate.head()

adult                              belongs_to_collection     budget  \
22058  False  {'id': 131635, 'name': 'The Hunger Games Colle...  130000000   
19814  False  {'id': 33514, 'name': 'The Twilight Collection...  120000000   
24873  False  {'id': 131635, 'name': 'The Hunger Games Colle...  125000000   
18150  False  {'id': 33514, 'name': 'The Twilight Collection...  110000000   
14425  False  {'id': 33514, 'name': 'The Twilight Collection...   50000000   

                                                  genres  \
22058  [{'id': 12, 'name': 'Adventure'}, {'id': 28, '...   
19814  [{'id': 12, 'name': 'Adventure'}, {'id': 14, '...   
24873  [{'id': 878, 'name': 'Science Fiction'}, {'id'...   
18150  [{'id': 12, 'name': 'Adventure'}, {'id': 14, '...   
14425  [{'id': 12, 'name': 'Adventure'}, {'id': 14, '...   

                                    homepage      id    imdb_id  \
22058       http://www.thehungergames.movie/  101299  tt1951264   
19814                                    NaN   50620  tt1673434   
24873       http://www.thehungergames.movie/  131631  tt1951265   
18150  http://www.breakingdawn-themovie.com/   50619  tt1324999   
14425       http://www.twilightthemovie.com/   18239  tt1259571   

      original_language                             original_title  \
22058                en            The Hunger Games: Catching Fire   
19814                en  The Twilight Saga: Breaking Dawn - Part 2   
24873                en      The Hunger Games: Mockingjay - Part 1   
18150                en  The Twilight Saga: Breaking Dawn - Part 1   
14425                en                The Twilight Saga: New Moon   

                                                overview        ...         \
22058  Katniss Everdeen has returned home safe after ...        ...          
19814  After the birth of Renesmee, the Cullens gathe...        ...          
24873  Katniss Everdeen reluctantly becomes the symbo...        ...          
18150  The new found married bliss of Bella Swan and ...        ...          
14425  Forks, Washington resident Bella Swan is reeli...        ...          

           revenue runtime                                   spoken_languages  \
22058  847423452.0   146.0           [{'iso_639_1': 'en', 'name': 'English'}]   
19814  829000000.0   115.0           [{'iso_639_1': 'en', 'name': 'English'}]   
24873  752100229.0   123.0           [{'iso_639_1': 'en', 'name': 'English'}]   
18150  712171856.0   117.0  [{'iso_639_1': 'en', 'name': 'English'}, {'iso...   
14425  709827462.0   130.0  [{'iso_639_1': 'en', 'name': 'English'}, {'iso...   

         status                                 tagline  \
22058  Released   Every revolution begins with a spark.   
19814  Released  The epic finale that will live forever   
24873  Released     Fire burns brighter in the darkness   
18150  Released           Forever is just the beginning   
14425  Released                The Next Chapter Begins.   

                                           title  video vote_average  \
22058            The Hunger Games: Catching Fire  False          7.4   
19814  The Twilight Saga: Breaking Dawn - Part 2  False          6.1   
24873      The Hunger Games: Mockingjay - Part 1  False          6.6   
18150  The Twilight Saga: Breaking Dawn - Part 1  False          5.8   
14425                The Twilight Saga: New Moon  False          5.6   

      vote_count belongs_to_series  
22058     6656.0             False  
19814     2641.0             False  
24873     5767.0             False  
18150     2622.0             False  
14425     2518.0             False  

[5 rows x 25 columns]

In [106]:
lionsgate.shape

(193, 25)

In [107]:
lionsgate.belongs_to_series.value_counts()

True     145
False     48
Name: belongs_to_series, dtype: int64

In [108]:
lionsgate = lionsgate.sort_values(by = 'revenue', ascending = False)
lionsgate = lionsgate.iloc[1:101,];
lionsgate.belongs_to_series.value_counts()

True     64
False    36
Name: belongs_to_series, dtype: int64

Out of all of the movies produced by Lionsgate Films, 24.9% of the movies belonged to a series. Out of the top 100 movies produced by Lionsgate Films, 36% of the movies belonged to a series.

## #8 The Weinstein Company

In [119]:
weinstein = prod_companies[prod_companies['production_companies'].str.contains("Weinstein|Lantern")];
weinstein.head()

adult belongs_to_collection     budget  \
20051  False                   NaN  100000000   
16129  False                   NaN   15000000   
13605  False                   NaN   70000000   
25202  False                   NaN   55000000   
19731  False                   NaN   21000000   

                                                  genres  \
20051  [{'id': 18, 'name': 'Drama'}, {'id': 37, 'name...   
16129  [{'id': 18, 'name': 'Drama'}, {'id': 36, 'name...   
13605  [{'id': 18, 'name': 'Drama'}, {'id': 28, 'name...   
25202  [{'id': 10751, 'name': 'Family'}, {'id': 35, '...   
19731  [{'id': 18, 'name': 'Drama'}, {'id': 35, 'name...   

                                        homepage      id    imdb_id  \
20051                 http://unchainedmovie.com/   68718  tt1853728   
16129                http://www.kingsspeech.com/   45269  tt1504320   
13605  http://www.inglouriousbasterds-movie.com/   16869  tt0361748   
25202                 http://www.paddington.com/  116149  tt1109624   
19731     http://silverliningsplaybookmovie.com/   82693  tt1045658   

      original_language           original_title  \
20051                en         Django Unchained   
16129                en        The King's Speech   
13605                en     Inglourious Basterds   
25202                en               Paddington   
19731                en  Silver Linings Playbook   

                                                overview        ...         \
20051  With the help of a German bounty hunter, a fre...        ...          
16129  The King's Speech tells the story of the man w...        ...          
13605  In Nazi-occupied France during World War II, a...        ...          
25202  A young Peruvian bear with a passion for all t...        ...          
19731  After spending eight months in a mental instit...        ...          

           revenue runtime                                   spoken_languages  \
20051  425368238.0   165.0  [{'iso_639_1': 'en', 'name': 'English'}, {'iso...   
16129  414211549.0   118.0           [{'iso_639_1': 'en', 'name': 'English'}]   
13605  319131050.0   153.0  [{'iso_639_1': 'de', 'name': 'Deutsch'}, {'iso...   
25202  259207227.0    95.0           [{'iso_639_1': 'en', 'name': 'English'}]   
19731  205738714.0   122.0           [{'iso_639_1': 'en', 'name': 'English'}]   

         status                                      tagline  \
20051  Released  Life, liberty and the pursuit of vengeance.   
16129  Released                             Find your voice.   
13605  Released  Once upon a time in Nazi occupied France...   
25202  Released                         The Adventure Begins   
19731  Released                          Watch For The Signs   

                         title  video vote_average vote_count  \
20051         Django Unchained  False          7.8    10297.0   
16129        The King's Speech  False          7.6     2817.0   
13605     Inglourious Basterds  False          7.9     6598.0   
25202               Paddington  False          7.0      899.0   
19731  Silver Linings Playbook  False          7.0     4840.0   

      belongs_to_series  
20051              True  
16129              True  
13605              True  
25202              True  
19731              True  

[5 rows x 25 columns]

In [120]:
weinstein.shape

(70, 25)

In [121]:
weinstein.belongs_to_series.value_counts()

True     62
False     8
Name: belongs_to_series, dtype: int64

Only 11.4% of the movies produced by the Weinstein Company were released as part of a series. 

## #9 Metro-Goldwyn-Mayer Studios

In [128]:
metro_goldwyn = prod_companies[prod_companies['production_companies'].str.contains("Goldwyn")];
metro_goldwyn.head()

adult                              belongs_to_collection     budget  \
19971  False  {'id': 121938, 'name': 'The Hobbit Collection'...  250000000   
22059  False  {'id': 121938, 'name': 'The Hobbit Collection'...  250000000   
25390  False  {'id': 121938, 'name': 'The Hobbit Collection'...  250000000   
893    False                                                NaN    4000000   
20536  False  {'id': 135468, 'name': 'G.I. Joe (Live-Action)...  130000000   

                                                  genres  \
19971  [{'id': 12, 'name': 'Adventure'}, {'id': 14, '...   
22059  [{'id': 12, 'name': 'Adventure'}, {'id': 14, '...   
25390  [{'id': 28, 'name': 'Action'}, {'id': 12, 'nam...   
893    [{'id': 18, 'name': 'Drama'}, {'id': 10749, 'n...   
20536  [{'id': 12, 'name': 'Adventure'}, {'id': 28, '...   

                        homepage      id    imdb_id original_language  \
19971  http://www.thehobbit.com/   49051  tt0903624                en   
22059  http://www.thehobbit.com/   57158  tt1170358                en   
25390  http://www.thehobbit.com/  122917  tt2310332                en   
893                          NaN     770  tt0031381                en   
20536  http://www.gijoemovie.com   72559  tt1583421                en   

                                  original_title  \
19971          The Hobbit: An Unexpected Journey   
22059        The Hobbit: The Desolation of Smaug   
25390  The Hobbit: The Battle of the Five Armies   
893                           Gone with the Wind   
20536                      G.I. Joe: Retaliation   

                                                overview        ...         \
19971  Bilbo Baggins, a hobbit enjoying his quiet lif...        ...          
22059  The Dwarves, Bilbo and Gandalf have successful...        ...          
25390  Immediately after the events of The Desolation...        ...          
893    An American classic in which a manipulative wo...        ...          
20536  Framed for crimes against the country, the G.I...        ...          

            revenue runtime                          spoken_languages  \
19971  1.021104e+09   169.0  [{'iso_639_1': 'en', 'name': 'English'}]   
22059  9.584000e+08   161.0  [{'iso_639_1': 'en', 'name': 'English'}]   
25390  9.560198e+08   144.0  [{'iso_639_1': 'en', 'name': 'English'}]   
893    4.001765e+08   238.0  [{'iso_639_1': 'en', 'name': 'English'}]   
20536  3.718763e+08   110.0  [{'iso_639_1': 'en', 'name': 'English'}]   

         status                                            tagline  \
19971  Released  From the smallest beginnings come the greatest...   
22059  Released  Beyond darkness... beyond desolation... lies t...   
25390  Released  Witness the defining chapter of the Middle-Ear...   
893    Released                  The greatest romance of all time!   
20536  Released                                                NaN   

                                           title  video vote_average  \
19971          The Hobbit: An Unexpected Journey  False          7.0   
22059        The Hobbit: The Desolation of Smaug  False          7.6   
25390  The Hobbit: The Battle of the Five Armies  False          7.1   
893                           Gone with the Wind  False          7.7   
20536                      G.I. Joe: Retaliation  False          5.4   

      vote_count belongs_to_series  
19971     8427.0             False  
22059     4633.0             False  
25390     4884.0             False  
893        995.0              True  
20536     3045.0             False  

[5 rows x 25 columns]

In [129]:
metro_goldwyn.shape

(1200, 25)

In [130]:
metro_goldwyn.belongs_to_series.value_counts()

True     1100
False     100
Name: belongs_to_series, dtype: int64

In [131]:
metro_goldwyn = metro_goldwyn.sort_values(by = 'revenue', ascending = False)
metro_goldwyn = metro_goldwyn.iloc[1:101,];
metro_goldwyn.belongs_to_series.value_counts()

True     63
False    37
Name: belongs_to_series, dtype: int64

Out of all of the movies produced by Metro-Goldwyn-Mayer Studios, 8.3% of the movies belonged to a series. Out of the top 100 movies produced by Metro-Goldwyn-Mayer Studios, 37% of the movies belonged to a series.

## #10 DreamWorks Pictures 

In [134]:
dreamworks = prod_companies[prod_companies['production_companies'].str.contains("DreamWorks|dreamworks")];
dreamworks.head()

adult                              belongs_to_collection     budget  \
7717   False  {'id': 2150, 'name': 'Shrek Collection', 'post...  150000000   
13833  False  {'id': 8650, 'name': 'Transformers Collection'...  150000000   
11825  False  {'id': 2150, 'name': 'Shrek Collection', 'post...  160000000   
15372  False  {'id': 2150, 'name': 'Shrek Collection', 'post...  165000000   
19051  False  {'id': 14740, 'name': 'Madagascar Collection',...  145000000   

                                                  genres  \
7717   [{'id': 12, 'name': 'Adventure'}, {'id': 16, '...   
13833  [{'id': 878, 'name': 'Science Fiction'}, {'id'...   
11825  [{'id': 14, 'name': 'Fantasy'}, {'id': 12, 'na...   
15372  [{'id': 35, 'name': 'Comedy'}, {'id': 12, 'nam...   
19051  [{'id': 16, 'name': 'Animation'}, {'id': 10751...   

                                            homepage     id    imdb_id  \
7717                          http://www.shrek2.com/    809  tt0298148   
13833              http://www.transformersmovie.com/   8373  tt1055369   
11825  http://www.shrekthethird.com/flash/index.html    810  tt0413267   
15372              http://www.shrekforeverafter.com/  10192  tt0892791   
19051                                            NaN  80321  tt1277953   

      original_language                       original_title  \
7717                 en                              Shrek 2   
13833                en  Transformers: Revenge of the Fallen   
11825                en                      Shrek the Third   
15372                en                  Shrek Forever After   
19051                en   Madagascar 3: Europe's Most Wanted   

                                                overview        ...         \
7717   Shrek, Fiona and Donkey set off to Far, Far Aw...        ...          
13833  Sam Witwicky leaves the Autobots behind for a ...        ...          
11825  The King of Far Far Away has died and Shrek an...        ...          
15372  A bored and domesticated Shrek pacts with deal...        ...          
19051  Alex, Marty, Gloria and Melman are still tryin...        ...          

           revenue runtime                                   spoken_languages  \
7717   919838758.0    93.0           [{'iso_639_1': 'en', 'name': 'English'}]   
13833  836297228.0   150.0  [{'iso_639_1': 'en', 'name': 'English'}, {'iso...   
11825  798958165.0    93.0           [{'iso_639_1': 'en', 'name': 'English'}]   
15372  752600867.0    93.0           [{'iso_639_1': 'en', 'name': 'English'}]   
19051  746921274.0    93.0  [{'iso_639_1': 'en', 'name': 'English'}, {'iso...   

         status                                            tagline  \
7717   Released                          Once upon another time...   
13833  Released                                 Revenge is coming.   
11825  Released                            Who's ready for Thirds?   
15372  Released                     It ain't Ogre... Til it's Ogre   
19051  Released  Six years ago, they disappeared without a trac...   

                                     title  video vote_average vote_count  \
7717                               Shrek 2  False          6.7     3082.0   
13833  Transformers: Revenge of the Fallen  False          6.0     3192.0   
11825                      Shrek the Third  False          6.0     2355.0   
15372                  Shrek Forever After  False          6.0     2021.0   
19051   Madagascar 3: Europe's Most Wanted  False          6.4     1857.0   

      belongs_to_series  
7717              False  
13833             False  
11825             False  
15372             False  
19051             False  

[5 rows x 25 columns]

In [135]:
dreamworks.shape

(155, 25)

In [136]:
dreamworks.belongs_to_series.value_counts()

True     129
False     26
Name: belongs_to_series, dtype: int64

In [137]:
dreamworks = dreamworks.sort_values(by = 'revenue', ascending = False)
dreamworks = dreamworks.iloc[1:101,];
dreamworks.belongs_to_series.value_counts()

True     77
False    23
Name: belongs_to_series, dtype: int64

Out of all of the movies produced by DreamWorks Pictures, 16.7% of the movies belonged to a series. Out of the top 100 movies produced by Dreamworks Pictures, 23% of the movies belonged to a series.